module-1/lab-numpy/your-code/main.py

```diff
@@ -198,29 +277,28 @@
         [ 'B',  'D',   'A',  'D', 'D']]])
 Again, you don't need Numpy in this question.
 """
-
-DIC = {0: "A", 
-       25: "B", 
-       50: "C", 
-       75: "D", 
-       100: "E"}
+DIC = {0.: "A", 
+       25.: "B", 
+       50.: "C", 
+       75.: "D", 
+       100.: "E"}
 
 res = []
 
-for i, x0 in enumerate(d): 
-    for j, x1 in enumerate(x0): 
-        for k, x2 in enumerate(x1): 
-            # print(f[i, j, k])
-            if d[i, j, k] == d_max: 
-                res.append(DIC[100])
-            elif d[i, j, k] == d_min: 
-                res.append(DIC[0])
-            elif d[i, j, k] == d_mean: 
-                res.append(DIC[50])
-            elif d_mean > d[i, j, k] > d_min: 
-                res.append(DIC[25])
-            else: 
-                res.append(DIC[75])
+for x in np.nditer(d): 
```

Comment:
Great!! Really good approach!!. There are at least another couple of ways of replacing values of an array without using loops. When you have some time, try it!! Remember to use Numpys official documentation and Saint Google and Stack Overflow.